regression/cbmc-primitives/dynamic-object-02/test-no-cp.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+CORE new-smt-backend
```

Comment:
Random place for comment: would it be possible to provide any indication which commit made things work?

Reply:
It's hard as it was the combination of 2 or more PRs.
Namely the one adding support for `byte_(update|extract)_exprt` and the dynamic memory slices.

I will write in the commit body, but I cannot link a specific commit/PR as there are multiple.